tests/test_unicode.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import unittest
+import inflect
+
+
+class TestUnicode(unittest.TestCase):
+    def test_unicode(self):
+        engine = inflect.engine()
+        # Unicode compatability test
```

Comment:
Typo: "compatability" -> "compatibility"

Reply:
Fixed in latest push